kytos/core/pacing.py

```diff
@@ -11,11 +11,51 @@
 LOG = logging.getLogger(__name__)
 
 
+class EmptyStrategy(limits.strategies.FixedWindowRateLimiter):
+    """Rate limiter, that doesn't actually rate limit."""
+
+    def hit(
+        self,
+        item: RateLimitItem,
+        *identifiers: str,
+        cost: int = 1
+    ) -> bool:
+        self.storage.incr(
```

Comment:
I had the impression that we just needed a no-op here. 

Can you elaborate on why do we need to increment the key?

Reply:
Lets you externally measure the rate at which the action is occurring. I could instrument this with logging to make this clearer.

Reply:
I see. Interesting. No need for extra logs. David, if you can add a code comment that suffice, just so whoever is reading this in the future understand why.

Reply:
Added!